specifyweb/frontend/js_src/lib/components/FormParse/__tests__/cells.test.ts

```diff
@@ -99,14 +99,15 @@ const cell = (
   cell: CellTypes[keyof CellTypes] &
     PartialBy<
       FormCellDefinition,
-      'align' | 'ariaLabel' | 'colSpan' | 'id' | 'visible'
+      'align' | 'ariaLabel' | 'colSpan' | 'id' | 'verticalAlign' | 'visible'
     >
 ): FormCellDefinition => ({
   id: undefined,
   colSpan: 1,
   align: 'left',
   visible: true,
   ariaLabel: undefined,
+  verticalAlign: 'stretch',
   ...cell,
 });
 
```

Comment:
Could you modify any of the tests in this file to have 2-3 tests that include verticalAlign?